casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala

```diff
@@ -203,7 +203,7 @@ object InterpreterUtil {
       _                  <- Span[F].mark("before-compute-parents-post-state-find-multi-parents")
       blockHashesToApply <- findMultiParentsBlockHashesForReplay(parents, dag)
       _ <- Log[F].info(
-            s"computeMultiParentsPostState computed number of parents: ${blockHashesToApply.length}"
+            s"replayIntoMergeBlock computed number of parents: ${blockHashesToApply.length}"
```

Comment:
While you are at it: I think it should say "computed number of uncommon ancestors".
"parents" is misleading as we were able to see while trying to understand the LPT issue.

Reply:
Done :)